less/navigation.less

```diff
@@ -233,6 +233,24 @@
   .discreet {
     .opacity(0.4);
   }
+
+  .navbar-primary.navbar-wrapped > .navbar-nav {
```

Comment:
These colour changes ought to be made in `.apply-navbar-brand()`of _less/mixins/branding.less_ as putting them here presumes white nav links (and that white will contrast with whatever brand colour is used).